android/app/src/main/java/com/microsoft/codepush/react/CodePushNativeModule.java

```diff
@@ -87,6 +87,11 @@ private boolean isReactApplication(Context context) {
     }
 
     private void loadBundleLegacy(final Activity currentActivity) {
+        if (currentActivity == null) {
```

Comment:
Since with your change we always pass `getCurrentActivity()` to this method.. I think it would be cleaner to just have `loadBundleLegacy` not take any method params, but simply call `getCurrentActivity()` here and check for nulls here.